src/nodes.coffee

```diff
@@ -3204,6 +3206,9 @@ exports.Op = class Op extends Base
     @operator in ['<', '>', '>=', '<=', '===', '!==']
 
   invert: ->
+    if @isIn()
```

Comment:
Until I read farther down, I thought this meant if is inverted.

Reply:
Ya this is a tricky name - updated to `@isInExpression()` but even that could be misinterpreted. `@isAnInExpression()`, though less ambiguous, looks weird to me

Reply:
Well doesn’t `in` refer to the operator? And this is in the `Op` class, so shouldn’t we call this `isInOperator`?

Reply:
@GeoffreyBooth sure updated to `isInOperator()`